runtime/config.go

```diff
@@ -37,15 +37,21 @@ const (
 
 // Config represents runtime configuration parameters
 type Config struct {
-	FirecrackerBinaryPath string `json:"firecracker_binary_path"`
-	KernelImagePath       string `json:"kernel_image_path"`
-	KernelArgs            string `json:"kernel_args"`
-	RootDrive             string `json:"root_drive"`
-	CPUCount              int    `json:"cpu_count"`
-	CPUTemplate           string `json:"cpu_template"`
-	LogLevel              string `json:"log_level"`
-	HtEnabled             bool   `json:"ht_enabled"`
-	Debug                 bool   `json:"debug"`
+	FirecrackerBinaryPath string       `json:"firecracker_binary_path"`
+	KernelImagePath       string       `json:"kernel_image_path"`
+	KernelArgs            string       `json:"kernel_args"`
+	RootDrive             string       `json:"root_drive"`
+	CPUCount              int          `json:"cpu_count"`
+	CPUTemplate           string       `json:"cpu_template"`
+	LogLevel              string       `json:"log_level"`
+	HtEnabled             bool         `json:"ht_enabled"`
+	Debug                 bool         `json:"debug"`
+	JailerConfig          JailerConfig `json:"jailer"`
+}
+
+// JailerConfig houses a set of configurable values for jailing
+type JailerConfig struct {
+	RuncBinaryPath string `json:"runc_binary_path"`
```

Comment:
1. Does the current schema allow us to specify network namepsace per VM? If not, can we please modify the interface to be able to do that? Doesn't need to be covered in this particular PR. We can follow it up in a separate issue as well.
2. Does the caller get a handle back to the cgroup hierarchies created by the jailer? I'm mostly asking because for VMs that require fractional CPUs, the orchestrator process would want to enforce these by modifying cpu quota & period settings for the vmm process
3. How does the caller specify the numa node for the VM?

Reply:
> Does the current schema allow us to specify network namepsace per VM? If not, can we please modify the interface to be able to do that? Doesn't need to be covered in this particular PR. We can follow it up in a separate issue as well.

Currently no. That'll be added as a feature later.

> Does the caller get a handle back to the cgroup hierarchies created by the jailer? I'm mostly asking because for VMs that require fractional CPUs, the orchestrator process would want to enforce these by modifying cpu quota & period settings for the vmm process

This does not currently return the cgroup path. We can add this as a feature request.

> How does the caller specify the numa node for the VM?

Currently no numa node can be specified. We've talked about this offline and have decided to be able to specify the CPUs and mem nodes as a better option.

Please see here for a list of what is to come, #258